src/app/game/services/example-drawing.service.ts

```diff
@@ -0,0 +1,22 @@
+import { endpoints } from '@/app/shared/models/endpoints';
+import { SupportedLanguages } from '@/app/shared/models/interfaces';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExampleDrawingService {
+  baseUrl = endpoints.TEKNISKBACKEND;
+
+  constructor(private http: HttpClient) {}
+
+  getExampleDrawings(numberOfDrawings: number, label: string, lang: SupportedLanguages): Observable<string[]> {
+    const body = '';

```

Comment:
Why do we send an empty string as the body?

Reply:
If I remember correctly, post requests made using the http-client from angular requires a body argument when made.

Reply:
I got an error when I tried to leave it out.

Reply:
Cleaned up with some more consistent structure here and in backend